Servers/Administration/OJS.Servers.Administration/Controllers/BaseAutoCrudAdminController.cs

```diff
@@ -58,6 +58,9 @@ protected TEntityId GetEntityIdFromQuery<TEntityId>(IDictionary<string, string>
         return (TEntityId)Convert.ChangeType(id, typeof(TEntityId));
     }
 
+    protected virtual Expression<Func<TEntity, bool>>? GetMasterGridFilter()
+        => this.MasterGridFilter;
```

Comment:
There are cases where the `MasterGridFilter` property returns a method `GetMasterGridFilter` invocation. Isn't this going to result in a recusive call? 

`Servers/Administration/OJS.Servers.Administration/Controllers/LecturersInContestsController.cs`

Reply:
GetMasterGridFilter returns an Expression with the invoked property. If the property is not one of the selected, the return value is null and nothing happens. This method is mainly created due to the virtual properties that were previously unable to filter with other properties of the chosen section.